x-pack/solutions/security/plugins/security_solution/public/common/lib/telemetry/events/telemetry_events.ts

```diff
@@ -15,6 +15,7 @@ import { manualRuleRunTelemetryEvents } from './manual_rule_run';
 import { notesTelemetryEvents } from './notes';
 import { onboardingHubTelemetryEvents } from './onboarding';
 import { previewRuleTelemetryEvents } from './preview_rule';
+import { siemMigrationsTelemetryEvents } from './siem_migrations';
 
 export const telemetryEvents = [
```

Comment:
As an aside, just realized we have no way of identifying if we have naming collisions in the telemetry event types. It shouldn't really ever happen since the naming is pretty well isolated, but in the future may be worth adding some check for that

Reply:
I added the test here https://github.com/elastic/kibana/pull/209633/commits/671771fbd78d94d03e9ab3e2c93824ab1ed10630